test/interpreter/function-vara.spec.ts

```diff
@@ -0,0 +1,43 @@
+import {ErrorType, HyperFormula} from '../../src'
+import {adr, detailedError} from '../testUtils'
+import {ErrorMessage} from '../../src/error-message'
+
+describe('Function VARA', () => {
+  it('should take at least two arguments', () => {
+    const engine = HyperFormula.buildFromArray([
+      ['=VARA()'],
+      ['=VARA(1)']
+    ])
+
+    expect(engine.getCellValue(adr('A1'))).toEqualError(detailedError(ErrorType.NA, ErrorMessage.WrongArgNumber))
+    expect(engine.getCellValue(adr('A2'))).toEqualError(detailedError(ErrorType.DIV_BY_ZERO))
+  })
+
+  it('should calculate variance (sample)', () => {
+    const engine = HyperFormula.buildFromArray([
+      ['=VARA(2, 3)'],
+    ])
+    expect(engine.getCellValue(adr('A1'))).toBeCloseTo(0.5, 6)
+  })
+
+  it('should coerce explicit argument to numbers', () => {
+    const engine = HyperFormula.buildFromArray([
+      ['=VARA(2, 3, 4, TRUE(), FALSE(), "1",)'],
+    ])
+    expect(engine.getCellValue(adr('A1'))).toBeCloseTo(2.28571428571429)
+  })
+
+  it('should ignore non-numeric values in ranges', () => {
```

Comment:
Same situtation with VARA and VARS as I've described for VARPS and VARPA.

```suggestion
  it('should evaluate TRUE to 1, FALSE to 0 and coerce text representation of numbers', () => {
```